modules/osdk-multi-arch-inspect-image-manifest.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-multi-arch-support.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-multi-arch-inspect-image-manifest_{context}"]
+= Inspecting your Operator's image manifest to find which architectures the workload images support
+
+Check which platforms your Operator can support by inspecting your image manifests. Image manifests specify the architectures and operating systems that your workload images support. You can inspect your Operator's image manifest by using build tools such as Skopeo or Docker.
+
+.Prerequisites
+
+* An Operator project
+* Skopeo or Docker installed
+
+.Procedure
+
+* Run the one of following commands to inspect your Operator's manifest and check which architectures are supported:
++
+[source,terminal]
+----
+$ skopeo inspect --raw <image>
+----
++
+[source,terminal]
+----
+$ docker manifest inspect <image>
+----
+where:
+<image>:: Specifies the Operator's image manifest, such as `docker://ubi9:latest`.
++
+.Example output
+[source,json]
+----
+{
+    "manifests": [
+        {
+            "digest": "sha256:c0669ef34cdc14332c0f1ab0c2c01acb91d96014b172f1a76f3a39e63d1f0bda",
+            "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+            "platform": {
+                "architecture": "amd64",
+                "os": "linux"
+            },
+            "size": 528
+        },
+...
+        {
+            "digest": "sha256:30e6d35703c578ee703230b9dc87ada2ba958c1928615ac8a674fcbbcbb0f281",
+            "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+            "platform": {
+                "architecture": "arm64",
+                "os": "linux",
+                "variant": "v8"
+            },
+            "size": 528
+        },
+...
+----
+
+.Troubleshooting
+
+* If the previous commands do not output platform information, then the specified image might be a single image instead of a manifest list. You can determine which architectures an image supports by running one of the following commands:
++
+[source,terminal]
+----
+$ skopeo inspect <image>
+----
++
+[source,terminal]
+----
+$ docker inspect <image>
```

Comment:
Verified :heavy_check_mark: 
<image> name when using docker for this command and manifest inspect should be `redhat/ubi9:latest`